test/integration/component/test_ldap_autoImport.py

```diff
@@ -0,0 +1,588 @@
+from marvin.cloudstackTestCase import cloudstackTestCase
```

Comment:
- Added License
- pep8 corrections incorporated
- Fixed the Filename
- ddt is not needed, removed it
- Requirements for this feature: AD server, and the details should be added to test_data.py.